src/arch/runtime/event_queue/iocp.hpp

```diff
@@ -1,12 +1,15 @@
 #ifndef ARCH_RUNTIME_EVENT_QUEUE_IOCP_HPP_
 #define ARCH_RUNTIME_EVENT_QUEUE_IOCP_HPP_
 
+#include <stdint.h>
```

Comment:
stdint.h is supported by POSIX and Windows so I don't see any potential for incompatibilities.  Using <cstdint> exports stuff in the std namespace so that's got a much higher cost of use.  Bad idea!